modules/ROOT/pages/anypoint-mq/anypoint-mq-connector-reference.adoc

```diff
@@ -115,14 +115,14 @@ Executes an Acknowledgement over a given AnypointMQMessageContext indicating tha
 | Acknowledgement Mode a| Enumeration, one of:
 
 ** IMMEDIATE
-** MANUAL |  Acknowledgement mode to use for the messages retrieved from this subscriber. Can only be 'MANUAL' or 'NONE'. |  MANUAL |
+** MANUAL |  Acknowledgement mode to use for the messages retrieved from this subscriber. Can  only be 'MANUAL' or 'NONE'. |  MANUAL |
```

Comment:
Can  only be 'MANUAL' or 'NONE'. |  MANUAL |

Extra space between Can and only